rules/prefer-add-event-listener.js

```diff
@@ -72,11 +75,31 @@ const create = context => {
 			codePathInfo = codePathInfo.upper;
 		},
 
+		Program() {
+			isDisabled = false;
```

Comment:
Why not just initialize the variable to `false`?

Reply:
Oh, I thought those variables were bound to the rule, so I wanted to reset `isDisabled` at the start of a new file 🤦‍♂️. That's what reviews are for 😄.